### ISSUE TITLE
Clean All failed chapters Error

### ISSUE THREAD

**Issue by eswiig** [oae/kaizoku#104]
### Describe the bug

![image](https://github.com/oae/kaizoku/assets/58608430/bb98f426-df18-4c37-9cff-ed447cfcdd73)

### Reproduction steps

Click Clear All Failed Chapters

### Expected behavior

To Bulk Clean All Failed Chapters

Cleaning Single Chapters Manually Works.

**Comment by Elryyyyyyyy** [comment]
I have this too why is there no response to this since may ? @oae

**Comment by ElryWeeb** [comment]
Yeah this project is a complete mess lol, massive bugs and flaws everywhere
